### PR TITLE
[CodeStyle] Use `pycon` for code-block language marker - part 20

### PR DESCRIPTION
**Pull request by enkilee**
### PR Category

User Experience

### PR Types

Docs

### Description

Replace docstring code-block language marker from python to pycon in some files where examples use interactive prompts (>>>).

### 是否引起精度变化

否

### CODE REVIEW

File: python/paddle/nn/functional/activation.py
Comment (SigureMo) [inline python/paddle/nn/functional/activation.py:1]: 怎么还开始和我家猫抢活了 :joy:，这个文件 #77656 在改

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -1974,44 +1989,35 @@ def ctc_loss(
...
             Tensor(shape=[2], dtype=float32, place=Place(cpu), stop_gradient=True,
-                   [3.91798496, 2.90765190])
+                   [3.91798496, 2.90765214])
 
-            >>> loss = F.ctc_loss(log_probs, labels,
-            ...     input_lengths,
-            ...     label_lengths,
-            ...     blank=0,
-            ...     reduction='mean')
+            >>> loss = F.ctc_loss(log_probs, labels, input_lengths, label_lengths, blank=0, reduction='mean')
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
            >>> loss = F.ctc_loss(log_probs, labels, input_lengths, label_lengths, blank=0, reduction='mean',)
```

File: python/paddle/nn/functional/loss.py [hunk 2]
```diff
@@ -2131,14 +2137,14 @@ def rnnt_loss(
...
-            ...      [0.1, 0.1, 0.2, 0.8, 0.1]],
-            ...     [[0.1, 0.6, 0.1, 0.1, 0.1],
-            ...      [0.1, 0.1, 0.2, 0.1, 0.1],
-            ...      [0.7, 0.1, 0.2, 0.1, 0.1]]
-            ... ]])
+            >>> acts = np.array(
+            ...     [
+            ...         [
+            ...             [[0.1, 0.6, 0.1, 0.1, 0.1], [0.1, 0.1, 0.6, 0.1, 0.1], [0.1, 0.1, 0.2, 0.8, 0.1]],
+            ...             [[0.1, 0.6, 0.1, 0.1, 0.1], [0.1, 0.1, 0.2, 0.1, 0.1], [0.7, 0.1, 0.2, 0.1, 0.1]],
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
            ...             [[0.1, 0.6, 0.1, 0.1, 0.1], [0.1, 0.1, 0.6, 0.1, 0.1], [0.1, 0.1, 0.2, 0.8, 0.1],],
            ...             [[0.1, 0.6, 0.1, 0.1, 0.1], [0.1, 0.1, 0.2, 0.1, 0.1], [0.7, 0.1, 0.2, 0.1, 0.1],],
```

File: python/paddle/nn/functional/loss.py [hunk 3]
```diff
@@ -2948,22 +2951,17 @@ def cross_entropy(
...
             >>> labels = paddle.uniform(shape, dtype='float64', min=0.1, max=1.0)
             >>> labels /= paddle.sum(labels, axis=axis, keepdim=True)
             >>> paddle_loss_mean = paddle.nn.functional.cross_entropy(
-            ...     logits,
-            ...     labels,
-            ...     soft_label=True,
-            ...     axis=axis,
-            ...     weight=weight,
-            ...     reduction=reduction
+            ...     logits, labels, soft_label=True, axis=axis, weight=weight, reduction=reduction
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
            ...     logits, labels, soft_label=True, axis=axis, weight=weight, reduction=reduction,
```

File: python/paddle/nn/functional/loss.py [hunk 4]
```diff
@@ -2975,33 +2973,24 @@ def cross_entropy(
...
 
             >>> # integer labels
             >>> paddle_integer_loss_mean = paddle.nn.functional.cross_entropy(
-            ...     logits,
-            ...     integer_labels,
-            ...     axis=axis,
-            ...     weight=weight,
-            ...     label_smoothing=label_smoothing,
-            ...     reduction=reduction
+            ...     logits, integer_labels, axis=axis, weight=weight, label_smoothing=label_smoothing, reduction=reduction
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
            ...     logits, integer_labels, axis=axis, weight=weight, label_smoothing=label_smoothing, reduction=reduction,
```

File: python/paddle/nn/functional/loss.py [hunk 5]
```diff
@@ -2975,33 +2973,24 @@ def cross_entropy(
...
 
             >>> # one_hot labels
             >>> paddle_one_hot_loss_mean = paddle.nn.functional.cross_entropy(
-            ...                                                         logits,
-            ...                                                         one_hot_labels,
-            ...                                                         axis=axis,
-            ...                                                         weight=weight,
-            ...                                                         label_smoothing=label_smoothing,
-            ...                                                         reduction=reduction)
+            ...     logits, one_hot_labels, axis=axis, weight=weight, label_smoothing=label_smoothing, reduction=reduction
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
            ...     logits, one_hot_labels, axis=axis, weight=weight, label_smoothing=label_smoothing, reduction=reduction,
```

File: python/paddle/nn/functional/loss.py [hunk 6]
```diff
@@ -4685,19 +4674,15 @@ def adaptive_log_softmax_with_loss(
             >>> tail_weights[0].append(paddle.randn([5, 2], dtype=paddle.float32))
             >>> tail_weights[0].append(paddle.randn([2, 1], dtype=paddle.float32))
             >>> out, loss = F.adaptive_log_softmax_with_loss(
-            ...     input,
-            ...     paddle.full([3], 1, dtype='int64'),
-            ...     head_weight,
-            ...     tail_weights,
-            ...     cutoffs=[2],
-            ...     head_bias=head_bias
+            ...     input, paddle.full([3], 1, dtype='int64'), head_weight, tail_weights, cutoffs=[2], head_bias=head_bias
```
Comment (SigureMo) [inline python/paddle/nn/functional/loss.py]:
```suggestion
            ...     input, paddle.full([3], 1, dtype='int64'), head_weight, tail_weights, cutoffs=[2], head_bias=head_bias,
```

File: python/paddle/nn/functional/norm.py [hunk 7]
```diff
@@ -207,14 +207,13 @@ def batch_norm(
             >>> weight = paddle.to_tensor([2], dtype="float32")
             >>> bias = paddle.to_tensor([1], dtype="float32")
 
-            >>> batch_norm_out = paddle.nn.functional.batch_norm(x, running_mean,
-            ...                                             running_variance, weight, bias)
+            >>> batch_norm_out = paddle.nn.functional.batch_norm(x, running_mean, running_variance, weight, bias)
```
Comment (SigureMo) [inline python/paddle/nn/functional/norm.py]:
```suggestion
            >>> batch_norm_out = paddle.nn.functional.batch_norm(x, running_mean, running_variance, weight, bias,)
```

File: python/paddle/nn/functional/conv.py [hunk 8]
```diff
@@ -421,20 +421,24 @@ def conv1d(
...
+            ...     ],
+            ...     dtype="float32",
+            ... )
+            >>> w = paddle.to_tensor(
+            ...     [
+            ...         [[9, 3, 4], [0, 0, 7], [2, 5, 6]],
+            ...         [[0, 3, 4], [2, 9, 7], [5, 6, 8]],
+            ...     ],
+            ...     dtype="float32",
+            ... )
```
Comment (SigureMo) [inline python/paddle/nn/functional/conv.py:453]:
```suggestion
            >>> x = paddle.to_tensor(
            ...     [
            ...         [[4, 8, 1, 9], [7, 2, 0, 9], [6, 9, 2, 6],],
            ...     ],
            ...     dtype="float32",
            ... )
            >>> w = paddle.to_tensor(
            ...     [
            ...         [[9, 3, 4], [0, 0, 7], [2, 5, 6],],
            ...         [[0, 3, 4], [2, 9, 7], [5, 6, 8],],
            ...     ],
            ...     dtype="float32",
            ... )
```

File: python/paddle/nn/functional/common.py [hunk 9]
```diff
@@ -2766,15 +2766,17 @@ class centers and the shape of sampled_class_center will be [num_positive_class_
         >>> num_classes_list = [10, 10]
         >>> num_classes = paddle.sum(paddle.to_tensor(num_classes_list))
         >>> label = paddle.randint(low=0, high=num_classes.item(), shape=[batch_size], dtype='int64')  # type: ignore[arg-type]
-        >>> label_list = [] # type: ignore
+        >>> label_list = []  # type: ignore
         >>> dist.all_gather(label_list, label)
         >>> label = paddle.concat(label_list, axis=0)
-        >>> remapped_label, sampled_class_index = paddle.nn.functional.class_center_sample(label, num_classes_list[rank_id], num_samples)
+        >>> remapped_label, sampled_class_index = paddle.nn.functional.class_center_sample(
+        ...     label, num_classes_list[rank_id], num_samples
```
Comment (SigureMo) [inline python/paddle/nn/functional/common.py]:
```suggestion
        ...     label, num_classes_list[rank_id], num_samples,
```

File: python/paddle/nn/functional/activation.py [hunk 10]
```diff
@@ -1244,21 +1247,22 @@ def softmax(
...
-            ...                       [[1.0, 2.0, 3.0, 4.0],
-            ...                        [5.0, 6.0, 7.0, 8.0],
-            ...                        [6.0, 7.0, 8.0, 9.0]]],dtype='float32')
+            >>> x = paddle.to_tensor(
+            ...     [
+            ...         [[2.0, 3.0, 4.0, 5.0], [3.0, 4.0, 5.0, 6.0], [7.0, 8.0, 8.0, 9.0]],
+            ...         [[1.0, 2.0, 3.0, 4.0], [5.0, 6.0, 7.0, 8.0], [6.0, 7.0, 8.0, 9.0]],
+            ...     ],
+            ...     dtype='float32',
+            ... )
```
Comment (SigureMo) [inline python/paddle/nn/functional/activation.py]:
```suggestion
            >>> x = paddle.to_tensor(
            ...     [
            ...         [[2.0, 3.0, 4.0, 5.0], [3.0, 4.0, 5.0, 6.0], [7.0, 8.0, 8.0, 9.0],],
            ...         [[1.0, 2.0, 3.0, 4.0], [5.0, 6.0, 7.0, 8.0], [6.0, 7.0, 8.0, 9.0],],
            ...     ],
            ...     dtype='float32',
            ... )
```

File: python/paddle/nn/functional/activation.py [hunk 11]
```diff
@@ -545,17 +545,20 @@ def prelu(
...
-            ...                            [ 3.0, -4.0,  5.0, -6.0],
-            ...                            [-7.0, -8.0,  8.0,  9.0]],
-            ...                           [[ 1.0, -2.0, -3.0,  4.0],
-            ...                            [-5.0,  6.0,  7.0, -8.0],
-            ...                            [ 6.0,  7.0,  8.0,  9.0]]]], dtype='float32')
+            >>> data = paddle.to_tensor(
+            ...     [
+            ...         [
+            ...             [[-2.0, 3.0, -4.0, 5.0], [3.0, -4.0, 5.0, -6.0], [-7.0, -8.0, 8.0, 9.0]],
+            ...             [[1.0, -2.0, -3.0, 4.0], [-5.0, 6.0, 7.0, -8.0], [6.0, 7.0, 8.0, 9.0]],
```
Comment (SigureMo) [inline python/paddle/nn/functional/activation.py]:
```suggestion
            ...             [[-2.0, 3.0, -4.0, 5.0], [3.0, -4.0, 5.0, -6.0], [-7.0, -8.0, 8.0, 9.0],],
            ...             [[1.0, -2.0, -3.0, 4.0], [-5.0, 6.0, 7.0, -8.0], [6.0, 7.0, 8.0, 9.0],],
```

File: python/paddle/nn/functional/activation.py
Comment (enkilee) [reply python/paddle/nn/functional/activation.py:1]: 那我这个文件不改了。

Comment (SigureMo) [reply python/paddle/nn/functional/activation.py:1]: 改吧，那边可以关掉

Comment (enkilee) [reply python/paddle/nn/functional/activation.py:1]: 刚还原提交了。交给猫猫

**Review by SigureMo — APPROVED** [review]
